subsys/bluetooth/audio/cap_internal.h

```diff
@@ -225,3 +275,19 @@ struct bt_cap_common_client *bt_cap_common_get_client_by_acl(const struct bt_con
 struct bt_cap_common_client *
 bt_cap_common_get_client_by_csis(const struct bt_csip_set_coordinator_csis_inst *csis_inst);
 int bt_cap_common_discover(struct bt_conn *conn, bt_cap_common_discover_func_t func);
+
+bool bt_cap_initiator_broadcast_audio_start_valid_param(
+	const struct bt_cap_initiator_broadcast_create_param *param);
+bool bt_cap_initiator_valid_unicast_audio_stop_param(
+	const struct bt_cap_unicast_audio_stop_param *param);
+int cap_initiator_unicast_audio_stop(const struct bt_cap_unicast_audio_stop_param *param);
+
+int cap_commander_broadcast_reception_start(
+	const struct bt_cap_commander_broadcast_reception_start_param *param);
```

Comment:
The naming here is a bit tricky. 

The `cap_initiator_unicast_audio_stop` and `cap_commander_broadcast_reception_start` were implemented as an internal calls to stop unicast audio and start the reception start without triggering the 

```
	if (bt_cap_common_test_and_set_proc_active()) {
		LOG_DBG("A CAP procedure is already in progress");

		return -EBUSY;
	}
``` 

checks. 

They cannot be prepended with `bt_` as that would conflict with the public functions. We could infix or suffix them with `internal` or just omit the `bt_`, but that doesn't really match the naming pattern we otherwise use for internal functions either.

Suggestions are welcome